apps/demo-app/app/components/NotificationCenter.vue

```diff
@@ -0,0 +1,205 @@
+<script lang="ts" setup>
+import {
+  iconBell,
+  iconBellRing,
+  iconCheckRead,
+  iconCircleAttention,
+  iconInbox,
+} from "@sit-onyx/icons";
+import {
+  OnyxAccordion,
+  OnyxAccordionItem,
+  OnyxBadge,
+  OnyxBottomBar,
+  OnyxEmpty,
+  OnyxHeadline,
+  OnyxNotificationCard,
+  OnyxSidebar,
+  useGlobalFAB,
+  useNotification,
+} from "sit-onyx";
+import { ref } from "vue";
+
+const store = useNotificationStore();
+const { t } = useI18n();
+
+const isSidebarOpen = ref(false);
+
+// simulate loading notifications
+const skeleton = ref(0);
+watch(
+  () => isSidebarOpen,
+  (isNewValue) => {
+    if (isNewValue) {
+      skeleton.value = store.unreadNotifications.length;
+      setTimeout(() => {
+        skeleton.value = 0;
+      }, 1000);
+    }
+  },
+);
```

Comment:
```suggestion
watch(isSidebarOpen, async (open) => {
  if (!open) return;
  skeleton.value = 3;
  await promiseTimeout(1000);
  skeleton.value = 0;
});
```

For this demo, I'd just always show the skeleton when opening the sidebar